test/powershell/Modules/Microsoft.PowerShell.Utility/Add-Type.Tests.ps1

```diff
@@ -1,6 +1,10 @@
 $guid = [Guid]::NewGuid().ToString().Replace("-","")
 
 Describe "Add-Type" -Tags "CI" {
+    It "Public 'Language' enumeration contains all members" {
+        [Enum]::GetNames("Microsoft.PowerShell.Commands.Language") -join "," | Should Be "CSharp,CSharpVersion7,CSharpVersion6,CSharpVersion5,CSharpVersion4,CSharpVersion3,CSharpVersion2,CSharpVersion1,VisualBasic,JScript"
+    }
+
```

Comment:
Two commits (one for code changes and one for tests) is not a strict requirement.